pallets/frequency-tx-payment/src/lib.rs

```diff
@@ -177,11 +177,25 @@ pub mod pallet {
 	#[pallet::call]
 	impl<T: Config> Pallet<T> {
 		/// Dispatch the given call as a sub_type of pay_with_capacity. Calls dispatched in this
-		/// fashion, if allowed, will pay with Capacity,
+		/// fashion, if allowed, will pay with Capacity
+		// The weight calculation is a temporary adjustment because overhead benchmarks do not account
+		// for capacity calls.  We count reads and writes for a pay_with_capacity call,
+		// then subtract one of each for regular transactions since overhead benchmarks account for these.
+		//	 Storage: Msa PublicKeyToMsaId (r:1)
+		//   Storage: Capacity CapacityLedger(r:1, w:2)
+		//   Storage: Capacity CurrentEpoch(r:1) ? maybe cached in on_initialize
+		//   Storage: System Account(r:1)
+		//   Total (r: 4-1=3, w: 2-1=1)
```

Comment:
Should this math match the values below in lines 193 and 195? I see 2 for reads, if I am understanding correctly.